api/services/registration_service.py

```diff
@@ -33,21 +33,42 @@ def handle_registration(self, jsonData):
                 )
             if flow_run_uuid:
                 call_log = models.CallLog.query.get_by_flow_run_uuid(flow_run_uuid)
+                user_program_data = {}
+                user_program_data["program_id"] = self.selected_program_id
                 if call_log:
                     registration_data = models.Registration.query.get_by_id(
                         call_log.registration_id
                     )
                     self.update_registration(registration_data, jsonData)
+                    user_program = (
+                        models.UserProgram.query.get_latest_active_user_program(
+                            self.user_id
+                        )
+                    )
```

Comment:
Lets move this outside the if condition. This way we don't have to write it in both `if` and `else` block

Reply:
Done.